web_ui/src/pages/annotator/notification/auto-training-credits-modal/util.ts

```diff
@@ -18,13 +18,16 @@ export const onFirstScheduledAutoTrainingJob =
 
         const { jobs, jobsCount } = pages[0];
         const totalScheduledJobs = Number(jobsCount.numberOfScheduledJobs);
-        const hasScheduleTrainingJobs = totalScheduledJobs > 0;
-        const neverAutotrained = getFuxSetting(FUX_SETTINGS_KEYS.NEVER_AUTOTRAINED, settings.config);
+        const totalRunningJobs = Number(jobsCount.numberOfRunningJobs);
+        const hasScheduledOrRunningTrainingJobs = totalScheduledJobs > 0 || totalRunningJobs > 0;
+        const neverAutoTrained = getFuxSetting(FUX_SETTINGS_KEYS.NEVER_AUTOTRAINED, settings.config);
         const isAutoTrainingJob = jobs.find((job) => {
-            return job.state === JobState.SCHEDULED && job.authorId === GETI_SYSTEM_AUTHOR_ID;
+            return (
+                (job.state === JobState.SCHEDULED || job.state === JobState.RUNNING) &&
```

Comment:
(Not for addressing now) – Just noticed the app relies on this conditional in multiple places. Since we already have some utilities in `shared/components/header/jobs-management/utils.ts`, maybe later we could add an `isJobRunningOrScheduled` helper to simplify things

```suggestion
                isJobRunningORScheduled(job.state) && job.authorId === GETI_SYSTEM_AUTHOR_ID
```